Source/Libraries/openXDA.Model/Channels/ChannelGroupType.cs

```diff
@@ -61,6 +65,10 @@ ChannelGroupType JOIN
 	        MeasurementCharacteristic ON ChannelGroupType.MeasurementCharacteristicID = MeasurementCharacteristic.ID
     ")]
     [SettingsCategory("systemSettings")]
+    [AllowSearch]
+    [PostRoles("Administrator, Transmission SME")]
+    [PatchRoles("Administrator, Transmission SME")]
+    [DeleteRoles("Administrator, Transmission SME")]
```

Comment:
```suggestion
```
All these attributes should be inherited by the child from the parent. Thus, re-adding it shouldn't be necessary.